apps/src/templates/progress/HiddenStageToggle.js

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
+import ProgressButton from './ProgressButton';
+import i18n from '@cdo/locale';
+
+const styles = {
+  main: {
+    wrap: 'nowrap',
+    marginTop: 5,
+    marginLeft: 15,
+    marginRight: 15
+  },
+  button: {
+    display: 'inline-block',
+    paddingLeft: 0,
+    paddingRight: 0,
+    boxSizing: 'border-box',
+    width: '50%'
+  },
+  leftButton: {
+    borderTopRightRadius: 0,
+    borderBottomRightRadius: 0,
+  },
+  rightButton: {
+    borderTopLeftRadius: 0,
+    borderBottomLeftRadius: 0,
+  },
+};
+
+const HiddenStageToggle = React.createClass({
```

Comment:
Might not be worth it yet, but I bet extracting a generic toggle component in this new style will be useful down the road.